.github/workflows/test-ruby.yml

```diff
@@ -0,0 +1,54 @@
+name: test-ruby
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  build-javascript:
+    uses: cucumber/html-formatter/.github/workflows/build-javascript.yml@github-ci
+
+  test-ruby:
+    needs: [build-javascript]
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        ruby: ['2.5', '2.6', '2.7', '3.0']
+        include:
+          - os: macos-latest
+            ruby: '3.0'
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Download javascript build artifacts
+      uses: actions/download-artifact@v2
+      with:
+        name: javascript-dist
+        path: javascript/dist
+
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler-cache: true
+        working-directory: ruby
+
+    - name: Copy javascript artifacts into assets folder
+      working-directory: ruby
+      run: |
+        cp ../javascript/dist/main.css assets/cucumber-html.css
+        cp ../javascript/dist/main.js assets/cucumber-html.js
+        cp ../javascript/dist/src/index.mustache.html assets/index.mustache.html
+
+    - run: bundle exec rake
```

Comment:
yay for delegating to `rake`!

Reply:
+1
But I won't prioritize that yet. It could be part of #3 actually